src/components/ArticleVotes.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import { patchArticleVotes } from "../../utils";
+import "../css/ArticleVotes.css";
+const ArticleVotes = ({ article }) => {
+  const { article_id, votes } = article;
+  const [votesChange, setVotesChange] = useState(0);
+  const incVotes = () => {
+    setVotesChange((currVotes) => currVotes + 1);
+    patchArticleVotes(article_id).catch((err) => {
+      setVotesChange((currVotes) => currVotes - 1);
+    });
+  };
+  const decVotes = () => {
+    setVotesChange((currVotes) => currVotes - 1);
+    patchArticleVotes(article_id).catch((err) => {
+      setVotesChange((currVotes) => currVotes + 1);
+    });
+  };
```

Comment:
There seem to be issues with this functionality when I tested it.
Downvoting seems to increase likes, the likes aren't shown after the buttons have been pressed - users need to reload the page. This is not great.
Also, we'd expect the votes to be optimistically rendered as well as there being some sort of message for the user if the vote can't currently be registered.